controllers/chat/ChatController.js

```diff
@@ -13,9 +13,133 @@ const BetterSocialConstantListUtils = require('../../services/bettersocial/const
 const {CHANNEL_TYPE} = require('../../helpers/constants');
 const ChatAnonUserInfoFunction = require('../../databases/functions/chatAnonUserInfo');
 const BetterSocialCore = require('../../services/bettersocial');
+const {Op} = require('sequelize');
+const crypto = require('crypto');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These are new import statements for `sequelize` and `crypto`. The `Op` object from `sequelize` is used for advanced queries, and `crypto` is used for generating a hash for the anonymous chat channel ID.
```javascript
const {Op} = require('sequelize');
const crypto = require('crypto');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->